### PR TITLE
add hullmods and corresponding graphics

### PR DESCRIPTION
**Pull request by Tsunder**

### CODE REVIEW

File: mod/data/hullmods/hull_mods.csv [hunk 1]
```diff
@@ -13,4 +13,6 @@ Order: Retreat when Missiles Expended,automatic_orders_retreat_no_missiles,0,,Au
...
-Order: No Retreat,automatic_orders_no_retreat,0,,Automatic Orders,,,0,TRUE,,,0,0,0,0,com.github.isturdy.automaticorders.hullmods.NoRetreat,Disables automatic retreat at low PPT/CR or high damage.,Disables automatic retreat at low PPT/CR or high damage.,graphics/automatic_orders/hullmods/no_retreat.png
\ No newline at end of file
+Order: No Retreat,automatic_orders_no_retreat,0,,Automatic Orders,,,0,TRUE,,,0,0,0,0,com.github.isturdy.automaticorders.hullmods.NoRetreat,Disables automatic retreat at low PPT/CR or high damage.,Disables automatic retreat at low PPT/CR or high damage.,graphics/automatic_orders/hullmods/no_retreat.png
+Order: Retreat at 10% PPT,automatic_orders_ten_ppt_retreat,0,,Automatic Orders,,,0,TRUE,,,0,0,0,0,com.github.isturdy.automaticorders.hullmods.RetreatTenPPT,Orders the ship to retreat when it reaches the 10% of its maximum Peak Performance Time. Overrides the default retreat threshold.,Orders the ship to retreat when it reaches 10% PPT threshold.,graphics/automatic_orders/hullmods/retreat_ten_ppt.png
```
Comment (Blothorn) [inline mod/data/hullmods/hull_mods.csv]: Nit: can you make these follow the "automatic_orders_retreat_{THRESHOLD}" convention?

**Review by Blothorn — CHANGES_REQUESTED** [review]
Thank you! Just one suggestion.

File: mod/data/hullmods/hull_mods.csv (hunk 1 above)
Comment (Tsunder) [reply mod/data/hullmods/hull_mods.csv]: as is tradition.
followed and resolved

**Comment by Tsunder** [comment]
addressed.
additionally, ordered hullmods to possibly follow order of descending readiness.

**Pull request merged**